### ISSUE TITLE
Couldn't find the library "topicmodels.utils"

### ISSUE THREAD

**Issue by rishabh170** [jwijffels/ceta-txtmining#1]
In the script "02_visualise_ceta_treaty.R" on line 17, the library is loaded, but I am not able to install this library. 

Also, could not find the function "document_frequency_matrix()" , possibly due to the above issue.

Please provide me detail how to get this library installed.

**Comment by jwijffels** [comment]
Currently package topicmodels.utils is not available in CRAN. It will be released in the coming months.
If you are looking for that package because you want to create a document/term matrix, please note you can also do this with a dozen of other packages on CRAN. If you are looking for making a word network correlation plot, you will have to wait a bit before it is released.

**Comment by rishabh170** [comment]
Thanks for replying. 

-Rishabh 

```
On Tuesday, 15 November 2016 9:42 PM, jwijffels <notifications@github.com> wrote:
```

 Currently package topicmodels.utils is not available in CRAN. It will be released in the coming months.
If you are looking for that package because you want to create a document/term matrix, please note you can also do this with a dozen of other packages on CRAN. If you are looking for making a word network correlation plot, you will have to wait a bit before it is released.—

**Comment by anurag173** [comment]
Hi! Is it available somewhere? I tried the termdocumentmatrix function from tm but then I'm not getting anything in the topterms array.

**Comment by jwijffels** [comment]
No, topicmodels.utils is currently not released, you can try the udpipe R package. It contains many of the topicmodels.utils functionalities.

**Comment by anurag173** [comment]
Okay I was able to get some topterms. But I'm still missing many functions like predict on LDA object, bareffects etc (I've installed udpipe). Can you please share topicmodels.utils?

**Comment by jwijffels** [comment]
As said topicmodels.utils is currently not released, you can try the udpipe R package. It contains many of the topicmodels.utils functionalities.
